core/src/main/java/org/springframework/security/scheduling/DelegatingSecurityContextTaskScheduler.java

```diff
@@ -30,47 +35,64 @@
  * @author Richard Valdivieso
  * @since 5.1
  */
-public class DelegatingSecurityContextTaskScheduler implements TaskScheduler {
+public class DelegatingSecurityContextTaskScheduler extends AbstractDelegatingSecurityContextSupport
+		implements TaskScheduler {
 
-	private final TaskScheduler taskScheduler;
+	private final TaskScheduler delegate;
```

Comment:
Since this variable name change is not necessary to close the ticket, would you please move this change to a separate commit in this PR?

Reply:
Moved to ba53e5d